angular-todo/e2e/centralized-login.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+
+test('centralized login angular', async ({ page }) => {
+  await page.goto('https://localhost:8443/home?centralLogin=true');
+
+  await page.getByRole('link', { name: 'Sign In', exact: true }).click();
+  await expect(page.getByText('Redirecting ...')).toBeVisible();
+
+  await page.getByLabel('User Name').fill('user01');
+  await page.getByLabel('Password').first().fill('Password1!');
+  await page.getByRole('button', { name: 'Next' }).click();
+
+  await page.waitForTimeout(3000);
```

Comment:
I dont think we should wait for timeout, its not a playwright best practice.

https://playwright.dev/docs/api/class-page#page-wait-for-timeout.

Reply:
Was there a conclusion on this?

Reply:
we had a small chat down below 
https://github.com/ryanbas21/forgerock-sdk-samples/pull/5#discussion_r1528629005

Reply:
Marking as resolved addressed here
https://github.com/ryanbas21/forgerock-sdk-samples/pull/5#discussion_r1528629005